library/core/src/Header.php

```diff
@@ -0,0 +1,223 @@
+     *     'datetimepicker',
+     *     'jquery-ui',
+     * ];
+     * ```
+     *
+     * The above example will render 2 `<script>` tags and 1 `<link>` tag which
+     * bring in the datetimepicker and jquery-ui versions defined in config.yaml
+     *
+     * @param array|string $assets Asset(s) to include
+     * @throws ParseException If unable to parse the config file
+     * @return string
+     */
+    static public function includeAsset($assets = [])
+    {
+
+        if (is_string($assets)) {
+            $assets = [$assets];
+        }
+
+        // @TODO Hard coded the path to the config file, not good RD 2017-05-27
+        $map = self::readConfigFile("{$GLOBALS['webroot']}/config/config.yaml");
+
+        $scripts = [];
+        $links = [];
+
+        foreach ($map as $k => $opts) {
+            $autoload = (isset($opts['autoload'])) ? $opts['autoload'] : false;
+            $rtl = (isset($opts['rtl'])) ? $opts['rtl'] : false;
+            if ($autoload === true || in_array($k, $assets)) {
+                $tmp = self::buildAsset($opts);
+
+                foreach ($tmp['scripts'] as $s) {
+                    $scripts[] = $s;
+                }
+
+                foreach ($tmp['links'] as $l) {
+                    $links[] = $l;
+                }
+
+                if ($rtl && $GLOBALS['language_direction'] == 'rtl') {
+                    $tmpRtl = self::buildAsset($rtl);
+                    foreach ($tmpRtl['scripts'] as $s) {
+                        $scripts[] = $s;
+                    }
+
+                    foreach ($tmpRtl['links'] as $l) {
+                        $links[] = $l;
+                    }
+                }
+            }
+        }
+
+        $linksStr = implode("", $links);
+        $scriptsStr = implode("", $scripts);
+        return "\n{$linksStr}\n{$scriptsStr}\n";
+    }
+
+    /**
+     * Build an html element from config options.
+     *
+     * @var array $opts Options
+     * @return array Array with `scripts` and `links` keys which contain arrays of elements
+     */
+    static private function buildAsset($opts = array())
+    {
+        $script = (isset($opts['script'])) ? $opts['script'] : false;
+        $link = (isset($opts['link'])) ? $opts['link'] : false;
+        $basePath = self::parsePlaceholders($opts['basePath']);
+
+        $scripts = [];
+        $links = [];
+
+        if ($script) {
+            $script = self::parsePlaceholders($script);
+            $path = self::createFullPath($basePath, $script);
+            $scripts[] = self::createElement($path, 'script');
+        }
+
+        if ($link) {
+            $link = self::parsePlaceholders($link);
+            $path = self::createFullPath($basePath, $link);
+            $links[] = self::createElement($path, 'link');
+        }
+
+        return ['scripts' => $scripts, 'links' => $links];
+    }
+
+    static private function loadTheme()
+    {
+        return self::createElement($GLOBALS['css_header'], 'link');
```

Comment:
This will break. Better to build this manually (just do what is in the scripts currently) than use createElement.
(this is because it deals with RTL and does the cache busting upstream)